src/routes/year/[year]/year.svelte

```diff
@@ -38,8 +39,8 @@
 			<div class="header-text-wrapper">
 				<Headline classes="green left">{data.year.event.title}</Headline>
 				<SubHeadline classes="year-header-subheadline white">
-					Online-Konferenz {formatDate(data.year.event.start_date, '%d.%m.')}
-					& {formatDate(data.year.event.end_date, '%d.%m.%y')}
+					Online-Konferenz {formatDate(data.year.event.start_date, Format.dateShort)}
```

Comment:
I don't know if it would be better to change it here or in `dates.ts`, but the short date should use a trailing dot. It should be "22.06. - 23.06.2024" instead of "22.06 - 23.06.2024".
```suggestion
					Online-Konferenz {formatDate(data.year.event.start_date, Format.dateShort).}
```